myadmin-service-api/src/main/java/za/ac/unisa/myadmin/qualification/services/StudentAcademicRecordService.java

```diff
@@ -0,0 +1,23 @@
+package za.ac.unisa.myadmin.qualification.services;
+
+import za.ac.unisa.myadmin.common.dto.ErrorInfo;
+import za.ac.unisa.myadmin.common.exceptions.DoesNotExistException;
+import za.ac.unisa.myadmin.common.exceptions.InvalidParameterException;
+import za.ac.unisa.myadmin.common.exceptions.MissingParameterException;
+import za.ac.unisa.myadmin.common.exceptions.OperationFailedException;
+import za.ac.unisa.myadmin.qualification.services.dto.StudentAcademicQualificationRecordInfo;
+
+import java.util.List;
+
+public interface StudentAcademicRecordService {
+
+	List<StudentAcademicQualificationRecordInfo> requestStudentAcademicQualificationResults(Integer studentNumber) throws MissingParameterException, InvalidParameterException, OperationFailedException, DoesNotExistException;
```

Comment:
Please refrain from naming methods "request..." if all you want to do is get a record. Just call it getStudentAcademicQualificationResults()

Reply:
Thanks, The other method requestStudentAcademicRecordEmail()
What should happen with it? Does a post make more sense?
ie. @Post and sendStudentAcademicRecordEmail()

Reply:
Oh so this is for sending an mail notification.
Yes make it a @Post operation then.

PS: Ideally I would like this to be in a separate MessageService that take a MessageInfo populated with the correct fields and values which can serve as the default service for all email communications. but Unisa is not making this easy and I think it will be too much work.